vuln/npm/409.json

```diff
@@ -0,0 +1,19 @@
+{
+  "id": 409,
+  "title": "merge-recursive prototype pollution",
+  "overview": "merge-recursive node module suffers from a prototype pollution vulnerability",
+  "created_at": "2018-02-17",
+  "updated_at": "2018-04-09",
+  "publish_date": "2018-04-09",
+  "author": "Olivier Arteau (HoLyVieR)",
+  "module_name": "merge-recursive",
+  "cves": [],
+  "vulnerable_versions": "",
+  "patched_versions": "",
+  "slug": "merge-recursive-prototype-pollution",
+  "recommendation": "No fix is currently available for this vulnerability.\n\nIt is our recommendation to not install or use this module at this time.",
+  "references": "- https://hackerone.com/reports/311337",
```

Comment:
report id is `311337`
am I the only one who finds it funny :-p

Reply:
Should I get a free beer ? 🍻